app/scraper.py

```diff
@@ -1,8 +1,15 @@
 from __future__ import print_function
-import os, json, sys
+import json
+import sys
 import requests
 from bs4 import BeautifulSoup
 
+try:
+  reload                        # Python 2
+except NameError:
+  from importlib import reload  # Python 3
```

Comment:
You can see how #148 has handled this. The way you have would be the right way. I think as of now, merging yours and making other PRs change their pulls accordingly would be the best thing to maintain compatibility.

Reply:
@Pk13055 says that we have to explicitly write out the best practice so I add it and then @vaibhavsingh97 wants me to take the best practice back out.  Please get together and decide which way you want it!